Assets/Scripts/Controllers/WorldController.cs

```diff
@@ -131,6 +131,7 @@ public void Start()
         spawnInventoryController.SetUIVisibility(Settings.GetSetting("DialogBoxSettings_developerModeToggle", false));
 
         cameraController.Initialize();
+        cameraController.Moved += this.World.OnCameraMoved;
 
```

Comment:
We would need to create the CameraController before the World. Maybe later, since we have things to improve in the event driven system.